docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+  postgres:
+    image: postgres
+    restart: always
+    ports:
+      - '5432:5432'
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
+      POSTGRES_DB: testing
+    volumes:
+      - postgres:/var/lib/postgresql/data
+volumes:
+  postgres:
```

Comment:
Why is this needed?

Reply:
For easier local setup. The Pg in the tests comes from actions service but when I want to set it up locally I need to translate that to a `docker-compose.yml` file.

Reply:
Root of the repository is not an appropriate location for this then as it gives a wrong impression about what these examples need or want to be run.

Reply:
Removed in https://github.com/prisma/prisma-examples/pull/2068